src/utils/webrtc/webrtc.js

```diff
@@ -935,10 +938,11 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		} else {
 			message = t('spreed', 'Error while accessing microphone & camera')
 			console.error('Error while accessing microphone & camera: ', error.message || error.name)
+			timeout = TOAST_DEFAULT_TIMEOUT
 		}
 
 		showError(message, {
-			timeout: TOAST_PERMANENT_TIMEOUT,
+			timeout: timeout,
```

Comment:
I dont think this should go away after 7sec. But it should close when you make a successful connection. For that we can store the return of showError in a variable and call variable.hide() or hideToast() or what the function is called, when a second request was successful

Reply:
Might apply to some messages, yes, but not for "Access to microphone & camera was denied"

Reply:
> Might apply to some messages, yes, but not for "Access to microphone & camera was denied"

But even that should immediatelly go away when you leave the call and join again and this time accept it. So save the return of `showError` in a var and in the beginning of this method check if `"var"` is not empty and then call the hideToast method, like in
https://github.com/nextcloud/spreed/blob/bd78555cc31e653171ec68a1f87eafe2304d7bba/src/services/conversationsService.js#L37-L50

Reply:
done